source/beamline/REIXS/REIXSXESMCPDetector.cpp

```diff
@@ -205,6 +211,24 @@ void REIXSXESMCPDetector::setTotalCountTarget(int totalCountTarget){
 		totalCountTarget_ = totalCountTarget;
 }
 
+void REIXSXESMCPDetector::pauseDwelling()
+{
+	if (dwellTimeTimer_) {
```

Comment:
If we don't have a dwell timer, do we still want to do this toggleVeto?  Seems like they would work together.